src/yaml.ts

```diff
@@ -196,4 +196,6 @@ export const loadString = (content: string | Buffer, filename: string): any =>
 export const dump = (doc: object): string =>
   jsyaml.safeDump(doc, {schema: schema, lineWidth: 999})
     .replace(/!<!([^>]+?)>/g, '!$1')
-    .replace(/ !\$include /g, ' !$ ');
+    .replace(/ !\$include /g, ' !$ ')
+    .replace(/\$0string \s*(0\d+)/g, "'$1'");
```

Comment:
Have you considered creating a custom tag for these "forced" strings? That way you can rely on the schema for dumping properly, rather than doing string replace

Reply:
Yeah, I tried that but it would require digging into the internals of jsyaml to wire it up correctly. There's no clean way of doing it without changes to jsyaml (or cloudformation).

Reply:
Specifically, jsyaml distinguishes between `implicit` and `explicit` tags. For a custom tag to work here it would have to be `implicit` and jsyaml doesn't directly expose that in an easy way.